README.md

```diff
@@ -37,3 +37,13 @@ MONITOR_TOKEN=[YOUR API TOKEN IN JENKINS]
 * Run the app by typing `rackup`
 
 It will fireup a web server and can be accessed at [localhost:4567](http://localhost:4567)
+
+## Docker  
+
+### Running  
+
+Running Jenkins Pipeline on Docker is simple as `docker run -p 4567:4567 -e MONITOR_USERNAME=<your-username> -e MONITOR_TOKEN=<your-token> -v <your-path-to-config>/jenkins-pipeline/config jenkinspipeline/jenkins-pipeline`
```

Comment:
Do we need that?
This is the docker way of life, I don't see a problem and a reason to not use this.
And also this is a very easy container image, there is some orchestrations which are way more complex than this.